experiences/homepage/homepage.test.js

```diff
@@ -70,6 +72,18 @@ const homepageTest = function() {
 
 		expect(isAtActiveSnapPoint).equal(true);
 	});
+
+	it('close card pressing close button', async()=>{
+		await scrollCard(browser, config.cards.homepage.cardSelector, 400);
+
+		await closeCard(browser);
+
+		const isAtInitialSnapPoint = await isAtSnapPoint(browser,
+			config.cards.homepage.cardSelector,
+			config.cards.homepage.snapPoints.initial);
+
+		expect(isAtInitialSnapPoint).equal(true);
```

Comment:
It would be nice to check the visibility of the button but we can do that in a different PR